rbac/management/group/view.py

```diff
@@ -225,6 +225,26 @@ def protect_system_groups(self, action, group=None):
             error = {key: [_(message)]}
             raise serializers.ValidationError(error)
 
+    def restrict_custom_default_group_renaming(self, request, group):
+        """Restrict users from changing the name or description of the Custom default group."""
+        method = request.method
+        name = request.data.get("name")
+        description = request.data.get("description")
+        invalid_methods = {"PUT"}
+        is_updating = method in invalid_methods
+        try:
+            custom_default_group = Group.objects.get(name="Custom default group")
+            is_custom_group = (
```

Comment:
you can use `group.platform_default and not group.system` to determine it is custom default group. You can